### ISSUE TITLE
`metric(0)` crashes

### ISSUE THREAD

**Issue by bersbersbers** [python-humanize/humanize#57]
### What did you do?

```shell
python -c "import humanize; humanize.metric(0)"
```

### What did you expect to happen?

No crash

### What actually happened?

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\bers\AppData\Local\Programs\Python\Python310\lib\site-packages\humanize\number.py", line 511, in metric
    exponent = int(math.floor(math.log10(abs(value))))
ValueError: math domain error
```

### What versions are you using?

* OS: Windows 10
* Python: 3.10.7
* Humanize: 4.3.0

Please include **code** that reproduces the issue.

```python
import humanize
humanize.metric(0)
```

**Comment by hugovk** [comment]
It's about time for a release, I've just released this in 4.4.0! 🚀 

https://github.com/python-humanize/humanize/releases/tag/4.4.0
https://pypi.org/project/humanize/4.4.0/

**Comment by bersbersbers** [comment]
Thank you ;)
